docs/content/direct/observability.md

```diff
@@ -12,12 +12,13 @@ KubeStellar controllers expose Prometheus-compatible metrics endpoints. These en
 | Controller                    | Protocol | Port  | Path     | AuthN/AuthZ | Notes |
 |-------------------------------|----------|-------|----------|-------------|-------|
 | kubestellar-controller-manager | HTTPS    | 8443  | /metrics | Kubernetes client authentication required when using the Service | Service: `kubestellar-controller-manager-metrics-service` (default port, configurable via Helm values). |
-| kubestellar-controller-manager | HTTPS    | 8443  | /metrics | None (direct pod access) | Access via Pod port 8443 (default, configurable via Helm values). |
+| kubestellar-controller-manager | HTTPS    | 8443  | /metrics | Kubernetes client authentication and authorization required even for Pod access | Access via Pod port 8443; HTTPS and authentication are always required. Default port, configurable via Helm values. |
```

Comment:
It is semi-redundant to repeat half of the authn/authz remark in the "Notes" column. It is enough to address authn and authz in the authn/authz column.